frontend/packages/ux-editor/src/components/FormComponent.tsx

```diff
@@ -61,14 +61,17 @@ export const FormComponent = memo(function FormComponent({
   const isPreviewable = previewableComponents.includes(component?.type as ComponentType);
 
   const handleDelete = (event: React.MouseEvent<HTMLButtonElement>): void => {
+    const confirmDelete = window.confirm(t("ux_editor.component_confirm_delete"));
```

Comment:
I think we should use the `Popover` component instead of `window.confirm`, as it's the component we use for the other confirmation dialogs. You can probably reuse a part of what you implemented in the text editor (`packages/text-editor/src/RightMenu.tsx`)

Reply:
done